.nais/valkey/dev.json

```diff
@@ -0,0 +1,5 @@
+{
+  "plan": "hobbyist",
+  "project": "nav-dev",
+  "prometheus": "f20f5b48-18f4-4e2a-8e5f-4ab3edb19733"
```

Comment:
Hvor kommer denne uuiden fra?

Reply:
Se under ServiceIntegration her: https://docs.nais.io/persistence/valkey/how-to/create-explicit/